src/sidebar/components/autocomplete-list.js

```diff
@@ -4,6 +4,24 @@ import { useMemo } from 'preact/hooks';
 import propTypes from 'prop-types';
 
 const defaultListFormatter = item => item;
+/**
```

Comment:
Style nit - Can you add a new line after `defaultListFormatter`.

Reply:
yep, thanks for pointing these out